client/src/component/ProfileWidget.js

```diff
@@ -2,133 +2,163 @@ import React from "react";
-                                <TextField 
-                                    id="outlined-basic" 
-                                    variant="outlined"
-                                    fullWidth={true}
-                                    InputProps={{
-                                        startAdornment: <InputAdornment position="start" className={classes.url_prefix}>calendapp.com/ <Divider className={classes.divider} orientation="vertical" /> </InputAdornment>,
-                                        defaultValue: "john-doe"
-                                    }}
-                                />
-                            </FormControl>
-                            </Box>
-                        </Grid>
-                    </Grid>
-                    <Grid container alignItems="center">
-                        <Grid item sm>
-                            <Typography alight="left" variant="subtitle2">
-                                {props.timezone_prompt}
-                            </Typography>
-                        </Grid>
-                        <Grid item sm={8}>
-                            <Box mr={3}>
-                            <FormControl variant="outlined" className={classes.formControl}>
-                                <InputLabel htmlFor="tz"></InputLabel>
-                                <Select
-                                    labelId="timezone-label"
-                                    id="tz"
-                                    value={timezone}
-                                    onChange={handleChange}>
-                                {/* 
+                            */}
+              <Box mr={3}>
+                <FormControl
+                  className={classes.url_input}
+                  noValidate
+                  autoComplete="off"
+                  fullWidth={true}
+                >
+                  <TextField
+                    id="outlined-basic"
+                    variant="outlined"
+                    fullWidth={true}
+                    InputProps={{
+                      startAdornment: (
+                        <InputAdornment
+                          position="start"
+                          className={classes.url_prefix}
+                        >
+                          calendapp.com/{" "}
+                          <Divider
+                            className={classes.divider}
+                            orientation="vertical"
+                          />{" "}
+                        </InputAdornment>
+                      ),
+                      defaultValue: "john-doe",
+                    }}
+                    onChange={handleChangeInUrl}
+                  />
+                </FormControl>
+              </Box>
+            </Grid>
+          </Grid>
+          <Grid container alignItems="center">
+            <Grid item sm>

```

Comment:
probably looking for this to be `sm={4}`

Reply:
fixed